internal/config/operations.go

```diff
@@ -0,0 +1,28 @@
+package config
+
+import (
+	"github.com/layer5io/meshery-adapter-library/adapter"
+	"github.com/layer5io/meshery-adapter-library/meshes"
+)
+
+var (
+	ServiceName = "service_name"
+)
+
+func getOperations(dev adapter.Operations) adapter.Operations {
+	versions, _ := getLatestReleaseNames(3)
+
+	dev[LinkerdOperation] = &adapter.Operation{
+		Type:        int32(meshes.OpCategory_INSTALL),
+		Description: "Linkerd Service Mesh",
+		Versions:    versions,
```

Comment:
Aren't versions and templates linked, ref our discussion a week ago, @kumarabd ?
Maybe it is the time to implement it now, before all adapters use it that way. Nevertheless, somewhere it should be documented which version we're talking about here.

Reply:
Hmm, this isn't updated with that either. Thanks for noticing @mgfeller , Ill think on a way to do that.

Reply:
I am not familiar with that discussion but does that mean that there should be different templates for different versions and hence we gotta know in advance that what versions we will be serving?

Reply:
Yes, let's continue the discussion here @kumarabd 

It wasn't (and still isn't) clear to me what version refers to _exactly_ and what it means.

So I assumed version refers to version of the workload specified by the set of templates: 
"default" version of something constitutes a set of one or several files, version 1.2 another set etc.

(also, if no version is specified in a request, it defaults to `default`)
I think that would be useful anyway.

But if version doesn't refer to that, or include that, then that's ok as well, of course.
It indicates that more documentation in the code about purpose and intention is needed, to disambiguate.

There are several versioned components in our code: adapter version, mesh version, workload version, operation version (maybe, what would that mean?).

Reply:
@mgfeller, just to clarify 
```go
	dev[LinkerdOperation] = &adapter.Operation{
		Type:        int32(meshes.OpCategory_INSTALL),
		Description: "Linkerd Service Mesh",
		Versions:    versions,
		Templates: []adapter.Template{
			"templates/linkerd.yaml",
		},
		AdditionalProperties: map[string]string{
			ServiceName: LinkerdOperation,
		},
	}
```
here version refers to a slice of latest releases of linkerd service mesh. This list is fetched from the github api when the adapter boots up.

Reply:
ok, thanks, yes, I saw that.
does this mean versions refers to the versions of the workload of the operation? if so, is there just one set of templates for all the different versions? 
or does this mean the versions of the mesh that support this workload? does it mean that one operation supports different versions of a workload, and the workload can be a mesh, or an application, or a configuration? different versions of something usually point to distinct releases of a binary, a template, a helm chart, a configuration etc. 

anyway, the design and intent should be documented close to the code, in this case in the meshery-adapter-library, I think. it shouldn't be necessary to go look at how it is used in different adapters, and infer intention from that. 

Does this clarify my question a bit, @kumarabd @utkarsh-pro ?